booking/models.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+import uuid
+
+
+class Event:
+
+    def __init__(self, title, total_ticket_count, ticket_price,
+                 sale_end, sale_start,
+                 max_tickets_in_order=50,
+                 order_expiration_period=timedelta(minutes=30)):
+        self._id = uuid.uuid4().hex
+        self.title = title
+        self.total_ticket_count = total_ticket_count
+        self.ticket_price = ticket_price
+        self.sale_start = sale_start
+        self.sale_end = sale_end
+        self.max_tickets_in_order = max_tickets_in_order
+        self.order_expiration_period = order_expiration_period
+
+    @property
+    def id(self):
+        return self._id
+
+
+class TicketOrder:
+
+    def __init__(self, event, ticket_count, ticket_holder_name,
+                 billing_address, order_open_time):
+        self._id = uuid.uuid4().hex
+        self.event = event
+        self.ticket_count = ticket_count
+        self.status = 0
```

Comment:
In case of status attribute there should be Enum used, which gives class for enumerated constants and is more readable and safer: https://docs.python.org/3.4/library/enum.html